e2e/select_test.go

```diff
@@ -402,9 +450,21 @@ func TestSelectCommand(t *testing.T) {
 				informat:      "csv",
 				compression:   true,
 				structure:     "\t",
-				outformat:     "csv",
+				outformat:     "json",
 				expectedValue: "{\"id\":\"id0\"}\n",
 			},
+			{
+				name: "input:csv,output:csv,delimiter:comma,all-versions:true,empty-bucket:true",
+				cmd: []string{
+					"select", "csv",
+					"--all-versions",
+					"--query", query,
+				},
+				informat:      "csv",
+				structure:     ",",
+				outformat:     "csv",
+				expectedValue: "-",
```

Comment:
I couldn't understand the reason for its use. Here, we write expectedValue as `-` and then convert the `-` string to an empty string ('') in the function below. Wouldn't it be the same if we used an empty string as the expected value instead? It is confusing

Reply:
Please let the reviewer decide whether the conversation is resolved or not, it's hard to track for the reviewer, every resolved conversation must be uncollapsed from the reviewers' side to see what was the response/comment.